frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/select-deployment.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { TFunction } from 'i18next';
+import { useTranslation } from 'react-i18next';
+import {
+  FormGroup,
+  Select,
+  SelectOption,
+  SelectProps,
+  SelectVariant,
+} from '@patternfly/react-core';
+
+import { DeploymentType } from '../../../../constants/create-storage-system';
+import { WizardDispatch, WizardState } from '../../reducer';
+import './backing-storage-step.scss';
+
+const options = [DeploymentType.FULL, DeploymentType.MCG];
+
+const optionsDescription = (t: TFunction) => ({
+  [DeploymentType.MCG]: t(
```

Comment:
we need to sentence case ` Multicloud Object Gateway ` right ?

Reply:
This was given by doc "MultiCloud Object Gateway"

Reply:
We should change it for consistency (either Full Deployment or Multicloud object gateway. Looks weird to me to have two different casing for two distinct items in a dropdown

Reply:
Okay got it.
Full deployment - is a general sentence
Multicloud Object Gateway - is the product name, so its fine.